collect_app/src/main/java/org/odk/collect/android/widgets/ImageWidget.java

```diff
@@ -141,7 +142,7 @@ private void hideButtonsIfNeeded(String appearance) {
 
     private void captureImage() {
         errorTextView.setVisibility(View.GONE);
-        if (selfie) {
+        if (selfie && new CameraUtils().isFrontCameraAvailable(getContext())) {
```

Comment:
I'm confused. You shouldn't need to do this because of the changes made in https://github.com/getodk/collect/pull/5344/commits/6bd11f65062b82184fc8b7c0ae23ed7c3bc59116 right?

Reply:
Ah wait, I see: you're just using the default camera app rather than my solution of falling back to the back camera in our custom implementation. That's much better. Maybe we should just revert the `CameraXCamera` changes in https://github.com/getodk/collect/commit/6bd11f65062b82184fc8b7c0ae23ed7c3bc59116?

Reply:
Why revert? If you do that then we will be blocked if the front camera is not available again. My change just complements yours. You removed blocking starting the camera when the front one is not present but there is `selfie` appearance and I improved that by using the default camera app instead of our implementation of CameraX.

Reply:
Sorry I don't think I was clear. I just meant removing the code [here](https://github.com/getodk/collect/blob/6bd11f65062b82184fc8b7c0ae23ed7c3bc59116/selfie-camera/src/main/java/org/odk/collect/selfiecamera/CameraXCamera.kt#L43) that falls back to the back camera if there's no front facing camera when we're using our custom implementation. As far as I can tell, we'll never be in that situation now that we use the device's camera app if there's no front camera.

Reply:
Ah ok I see. Yes we can remove it what I've just done.